cmd/gapir/cc/main.cpp

```diff
@@ -57,27 +57,32 @@
 using namespace core;
 using namespace gapir;
 
+const size_t RAM_CACHE_SIZE = 256ul * 1024ul * 1024ul;
```

Comment:
My understanding is that this is used for volatile & cache memory. This is not nearly big enough for many Vulkan traces on Desktop. 3GB is the minimum we  could get away with. It would maybe be good to start this at 4GB and work our way down.

Reply:
Is that safe for 32 bit architectures? It will overflow a size_t. Should we have two starting figures for 32 and 64 bit archs respectively? Then recursively half until we get something we can allocate?

Reply:
Yep that is fine with me.
4GB for 64-bit , and 2GB for 32-bit would be fine.

Reply:
To be honest, we could also do more for 64-bit. 8GB to start maybe?

Reply:
Why not go large with 16gb? or even 64? Until we touch the physical pages they shouldn't cost anything on VM based systems. Unless LMK works off virtual and not physical usage?